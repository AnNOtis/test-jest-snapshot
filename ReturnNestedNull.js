import React from 'react'
import ReturnNull from './ReturnNull'

const ReturnNestedNull = () => <ReturnNull />

export default ReturnNestedNull
