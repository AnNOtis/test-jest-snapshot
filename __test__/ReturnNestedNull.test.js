import React from 'react';
import { shallow } from 'enzyme';

import ReturnNestedNull from '../ReturnNestedNull';

describe('<ReturnNestedNull />', () => {

  it('should render', () => {
    const component = shallow(<ReturnNestedNull />);

    expect(component).toMatchSnapshot();
  });
});
