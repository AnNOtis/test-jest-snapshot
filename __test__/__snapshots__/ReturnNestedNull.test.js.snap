// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReturnNestedNull /> should render 1`] = `<ReturnNull />`;
